website/blog/2022-03-03-adding-blocks-wpcli.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Adding components and blocks with WP-CLI
+description: This blog post covers how to use our WP CLI commands for adding components and blocks into our projects.
+slug: adding-blocks-wpcli
+authors: iobrado
+date: 2022-03-10
+tags: [eightshift, boilerplate, wpcli, components, blocks]
+hide_table_of_contents: false
+---
+Although there are a few basic blocks available after installing the Eightshift theme, there are a lot more blocks available, but you have to add them to your project with WP-CLI. To see the complete list of available components and blocks, please visit our [Storybook](https://infinum.github.io/eightshift-docs/storybook/). 
```

Comment:
A suggestion for rephrasing the intro a bit - feel free to disregard if you don't agree. 

```suggestion
Although there are a few basic blocks available after creating a project, there are a lot more blocks available. However, you have to add them to your project with WP-CLI. To see the complete list of available components and blocks, visit our [Storybook](https://infinum.github.io/eightshift-docs/storybook/). 
```

Reply:
This is actually much better 😄